cmd/operator/main.go

```diff
@@ -55,14 +59,18 @@ func init() {
 type devicePluginControllerAndWebhook map[string](func(ctrl.Manager, string, bool) error)
 
 func main() {
+	var plugins = []string{"dsa", "fpga", "gpu", "qat", "sgx"}
 	var metricsAddr string
 	var devicePluginNamespace string
 	var enableLeaderElection bool
+	var deviceType string
+	var pm *patcher.PatcherManager
 	flag.StringVar(&metricsAddr, "metrics-addr", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&devicePluginNamespace, "deviceplugin-namespace", metav1.NamespaceSystem, "The namespace where deviceplugin daemonsets are created")
 	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.StringVar(&deviceType, "device", "", "Device type to set up.")
```

Comment:
does this mean that only one device can be chosen?

Reply:
Sure, we can allow multiple devices. However as kustmizations overlays are per device type, would it make sense, what do you think?

Reply:
I don't know if this is possible, but it would be nice to support more than one device at a time. Otherwise if cluster has more than one device type users will have to use default operator with support of all devices.

Reply:
OK, let's support also more then one device at a time.

Reply:
Sure, let's make it so.

Reply:
I'd suggest a bit more user friendly approach:
```
package main
  
import "fmt"
import "flag"
import "strings"

type flagList []string

func (fl *flagList) String() string {
        return strings.Join(*fl, ", ")
}

func (fl *flagList) Set(value string) error {
        *fl = append(*fl, value)
        return nil
}

var devices flagList

func main() {
        flag.Var(&devices, "device", "device to enable, can be used multiple times")
        flag.Parse()
        fmt.Println(devices)
}
```

Reply:
sorry, forgot to show how it looks in the command line:
```
$ go run flags.go --device fpga --device gpu
[fpga gpu]
```

Reply:
OK, thanks, let's do so.

Reply:
+1